src/components/Garden/Preferences/EVMExecutor.js

```diff
@@ -57,6 +62,14 @@ function EVMExecutor() {
     getEvmCrispr()
   }, [account, connectedGarden, ethers])
 
+  const forwarderName = useMemo(() => {
+    if (!gardenState || !gardenState.installedApps) {
+      return null
+    }
+    return getAppByName(gardenState.installedApps, env('VOTING_APP_NAME'))
+      .artifact.appName
```

Comment:
EVMcrispr do not understand the full `appName`. Let's do `appName.split('.aragonpm.eth')[0]`. To only keep the first part.